tests/apps/Dockerfile-windows

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License.
 # ------------------------------------------------------------
 
-FROM golang:1.15-nanoserver-1809
+FROM mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
I tried this and it was missing a lib that golang needed, I think? Have you run this yourself?

Reply:
hmm if weadd static link option, then it should work without dependency. let me check

Reply:
Disabled CGO_ENABLED, which will link standard lib statically.

Reply:
Ah, ok, though it looks like it worked anyway. I don't remember what specifically it was that made me run into this problem.